app/routes/image-set/index.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  controllerName: 'image-set',
+
+  model: function() {
+    return this.modelFor('imageSet');
```

Comment:
I believe that the default behavior is for this route to inherit it's parent's model. This code may not be necessary